osutil/overlay_linux.go

```diff
@@ -69,6 +69,16 @@ func IsRootWritableOverlay() (string, error) {
 					continue
 				}
 
+				if dir == "/media/root-rw/overlay" {
+					// On the Ubuntu server ephemeral image, '/' is setup via
```

Comment:
It may help you to have a more specific comment here.

It's not "Ubuntu server" that does this.
Its 'overlayroot' (package) that sets things up this way.

overlayroot package is configured by kernel command line or /etc/overlayroot.conf or /etc/overlayroot.local.conf in all supported Ubuntu.  The package is installed by default in Ubuntu Server but is available in Ubuntu since 12.04.

Reply:
Also just note that the 'overlay' portion of '/media/root-rw/overlay' is configurable via overlayroot.conf or kernel command line.  overlayroot=tmpfs:dir=/mydir .  dir defaults to '/overlay'.